service/db/import.py

```diff
@@ -296,6 +298,21 @@ async def process_chain(
     logger.info(f"Imported {n_new_prices} new prices for {code}")
 
 
+async def import_archive(path: Path):
+    """Import data from all chain directories in the given zip archive."""
+    try:
+        price_date = datetime.strptime(path.stem, "%Y-%m-%d")
+    except ValueError:
+        logger.error(f"`{path.stem}` is not a valid date in YYYY-MM-DD format")
+        return
+
+    with TemporaryDirectory() as temp_dir:
```

Comment:
Seems legit but `ty` gives out a spurious error here (see build failures), I'm fine with just adding `# type: ignore` for now.